exercises/practice/diamond/.meta/example.cpp

```diff
@@ -0,0 +1,25 @@
+#include "diamond.h"
+
+namespace diamond {
+    std::vector<std::string> rows(char middle_letter) {
+        if (middle_letter == 'A') return {"A"};
+        std::vector<std::string> diamond;
+        for(char c{'A'}; c <= middle_letter; ++c) {
+            int outer_spacing = middle_letter - c;
+            std::string row{};
+            row.append(std::string(outer_spacing, ' '));
```

Comment:
Did you know that `std::string::append()` has an overload that takes a count and a `char`? That would shorten these `append` lines a little bit.

Reply:
Good catch, coding felt clunky, but I did not have that in my toolbelt to make it better.